docs/semgrep-app/pricing-and-billing.mdx

```diff
@@ -127,13 +135,13 @@ Notes:
 
 ## Determining your plan needs
 
-### Number of seats
+### Number of developers
 
-Within your team or organization, assess the number of members whose code needs scanning. That will determine the number of seats needed for purchase.
+Within your team or organization, assess the number of members that make commits. That determines the number of developers needed for the plan purchase.
 
-For example, if a project has 4 unique contributors who push code during the billing period and Semgrep runs on their merge or pull requests, only 4 seats are required even if the organization has a total of 10 members. If these unique contributors commit to many projects within the same organization, they will only be counted once, so no additional seats are required.
+For example, if a project has 4 unique developers who create commits during the billing period and Semgrep scans their repositories, only 4 developers are required even if the organization has a total of 10 members. If these unique developers commit to many projects within the same organization, they will only be counted once, so no additional cost is incurred.
```

Comment:
(Again feel free to ignore and resolve) I guess incur meaning is obvious, but it seems to be a rather C2 word (see [dictionary](https://www.merriam-webster.com/dictionary/incur)), probably fine for natives and advanced English speakers. In case you want to simplify it, maybe:
```suggestion
For example, if a project has 4 unique developers who create commits during the billing period while Semgrep is scanning their repositories, only 4 developers are required even if the organization has a total of 10 members. If these unique developers commit to many projects within the same organization, they are counted once, so no additional cost is charged.
```